.changeset/ready-rice-grin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@shopware/composables": minor
+---
+
+Removed `setDefaultPaymentMethod` method from `useUser` composable
```

Comment:
that is a breaking change, few things needed:

- since which version this endpoint is removed
- consider deprecating that endpoint if someone is using older version with notification that this will be removed in next version
- inform what user should do instead of this endpoint